tests/Unit/Framework/SetupLocalPythonEnvironment.cpp

```diff
@@ -63,7 +64,7 @@ SetupLocalPythonEnvironment::SetupLocalPythonEnvironment(
 }
 
 #if PY_MAJOR_VERSION == 3
-std::nullptr_t SetupLocalPythonEnvironment::init_numpy() {
+static std::nullptr_t SetupLocalPythonEnvironment::init_numpy() {
```

Comment:
This is probably my fault: you need to mark it static in the `hpp` only... And should probably do it for both the 3 and 2 versions